content/correspondence/reference/API endpoints/initialize/_index.en.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Initialize
+linktitle: Initialize
+description: Endpoint for initializing a correspondence.
+
+weight: 60
+toc: true
+---
+
+## Endpoint
+
+POST /correspondence/api/v1/correspondence
```

Comment:
Maybe add a general description here which explains the business logic of the endpoint. Worth mentioning that it requires attachment to be upload before this